src/security/initializable.cairo

```diff
@@ -1,6 +1,10 @@
 // SPDX-License-Identifier: MIT
 // OpenZeppelin Contracts for Cairo v0.7.0 (security/initializable.cairo)
 
+trait ExternalTrait<TState> {
+    fn is_initialized(self: @TState) -> bool;
+}
```

Comment:
No strong opinions on including this trait. I suppose there may be cases where this is useful

Reply:
The name IPausable sounds a bit misleading IMO (for an interface with just the `is_paused` method), I think for this case is better to use a trait (like PublicKeyTrait in account), I selected the name ExternalTrait because no better name occurred to me for just is_paused. We should document this convention/pattern if we go along with it. What do you think @martriay ?

Reply:
> The name IPausable sounds a bit misleading IMO (for an interface with just the is_paused method)

Good point. Here, it'd be `IInitializable`, but the point is the same. I can see it being misleading with just the getter

Reply:
> We should document this convention/pattern if we go along with it.

I just [wrote this](https://github.com/OpenZeppelin/cairo-contracts/issues/696#issuecomment-1803235090) recently in our style guide, which is something I recall we discussed maybe during a call. Let me know if you think it doesn't apply here

> The name IPausable sounds a bit misleading IMO (for an interface with just the `is_paused` method)

I kindof see what you mean, but on the other hand "pausable" doesn't really mean I can pause it, but rather that it can be paused, so the getter is not really confusing to me. btw Contracts for Solidity's version has [public pause/unpause methods](https://github.com/ConsenSysMesh/openzeppelin-solidity/blob/master/contracts/lifecycle/Pausable.sol#L37-L48)